src/cbc_sdk/audit_remediation/base.py

```diff
@@ -588,11 +845,50 @@ def submit(self):
 
         return self._doc_class(self._cb, initial_data=resp.json())
 
+    def _init_async_query(self):
+        """
+        Initialize an async query and return a context for running in the background. Optional.
+
+        Returns:
+            Run: Context for running in the background.
+        """
+        return self.submit()
+
+    def _run_async_query(self, context):
```

Comment:
I'm not sure this makes sense given the time for a run to complete.

Reply:
Hmmm...not sure what I can say to that. The functionality is there, and it works in unit tests.